client/src/pages/home/HomePage.tsx

```diff
@@ -12,16 +15,36 @@ const makePhrases = async (len: number) => {
     userFlavors.push({
       name: phrase,
       price: 0,
-      image: "/strawberry-icecream.jpg",
+      imageURL: "/strawberry-icecream.jpg",
```

Comment:
Josh mentioned that Hugging Face image loading takes a long time. so it'd load for too long to generate 5 images every time somebody refreshes the homepage. We were considering removing this and instead extending the "5 existing user-generated flavors" to 10 (2 rows of 5) so that there are more options to choose from.

but we don't want to make that decision ourselves, especially given how much effort @pearlescence-m put into generating adj+noun on the front page.

does the team have any suggestions with how to proceed?

Reply:
Ok! How long are we talking here? Could this be a good place for progressive image loading or lazy loading?


https://blog.logrocket.com/progressive-image-loading-react-tutorial/
https://web.dev/articles/lazy-loading-images